README.md

```diff
@@ -20,13 +20,34 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
 * [TensorFlow 1.0 or 1.1](https://www.tensorflow.org/install/)
 * [SciPy](https://scipy.org/install.html)
 * [PyXDG](https://pypi.python.org/pypi/pyxdg)
-* [python_speech_features](https://pypi.python.org/pypi/python_speech_features)
```

Comment:
I think we should leave this here during the deprecation period, that way it works out of the box even if libdeepspeech isn't installed.